Tests/PackageLoadingTests/PD4_2LoadingTests.swift

```diff
@@ -516,17 +516,12 @@ class PackageDescription4_2LoadingTests: PackageDescriptionLoadingTests {
         )
         """
 
-        try loadManifestThrowing(stream.bytes) { manifest in
-            if let dep = manifest.dependencies.first {
-                switch dep {
-                case .scm(let scm):
-                    XCTAssertEqual(scm.location, "/best")
-                default:
-                    XCTFail("dependency was expected to be remote")
-                }
-            } else {
-                XCTFail("manifest had no dependencies")
-            }
+        do {
```

Comment:
nit: use XCTAssertThrowsError?

Reply:
There is even `XCTAssertManifestLoadThrows`